src/Spice86.Core/Emulator/CPU/CfgCpu/ParsedInstruction/Instructions/Aaa.cs

```diff
@@ -24,4 +25,8 @@ public override void Execute(InstructionExecutionHelper helper) {
         helper.State.CarryFlag = finalCarryFlag;
```

Comment:
Undirect dependency. Maybe have a _state class variable ?

Reply:
Instructions are not supposed to store state